extensions/widgets/graph/notes/Add optional decimal to Y-Labels.md

```diff
@@ -0,0 +1 @@
+Y-Labels can optionally display decimaals
```

Comment:
@BerndN I think the filename should be something like `feature-decimal-y-labels.md` and it looks like you missed a typo in `decimals` here.